routes/topics.js

```diff
@@ -30,7 +30,7 @@ router.post(
   topicsController.inviteMembers
 );
 router.post('/create', validate(TopicValidation.create), isAuth, topicsController.createTopic);
-router.get('/', isAuth, topicsController.getTopics);
+router.get('/', validate(TopicValidation.getTopic), isAuth, topicsController.getTopics);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change is good as it adds validation to the `get` route for topics. However, it's important to ensure that the `TopicValidation.getTopic` schema includes all necessary validations for the query parameters, including the new `withMinimumFollower` parameter. If not already done, this should be a number and possibly have a minimum value set to prevent misuse.
```diff
- router.get('/', validate(TopicValidation.getTopic), isAuth, topicsController.getTopics);
+ router.get('/', validate(TopicValidation.getTopicWithMinFollowers), isAuth, topicsController.getTopics);
```
In the above diff, `TopicValidation.getTopicWithMinFollowers` should be a new validation schema that validates all the query parameters including `withMinimumFollower`.


<!-- This is an auto-generated comment by OSS CodeRabbit -->